website/docs/validating-admission-policy

```diff
@@ -0,0 +1,71 @@
+---
+id: validating-admission-policy
+title: Integration with Kubernetes Validating Admission Policy
+---
+
+`Feature State`: Gatekeeper version v3.13+ (pre-alpha)
```

Comment:
it would be good to align on a language and add to our release process. we call this `experimental` in the flags, do we want to call it the same here?

Reply:
For feature states, we have been using alpha, beta, etc. Pre-alpha seems closer to alpha. I'm ok either way. LMK WYT.

Reply:
We don't have to solve this in this PR. As long as we make a decision and document (this came up a few times), that sgtm